data/mods/Aftershock/itemgroups/vending_machine_group.json

```diff
@@ -19,9 +19,10 @@
     "type": "item_group",
     "subtype": "collection",
     "id": "afs_vending_ballistic_ammo",
+    "//": "Small metal box is a placeholder until aftershock specific ammo boxes are made",
```

Comment:
You can remove this comment, individual ammo boxes is several more layers of simulations that what I care to simulate in aftershock.

Reply:
Would you at least want generic ammo boxes?

Reply:
Sure a single ammo box type item seems fine.